lib/diagnostic.rb

```diff
@@ -121,15 +121,22 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+
+  def roar
+    puts "ROAR"
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# Using mixins is more popular than inheritance because you don’t have to think
+# about the relationship between classes.
 ##
```

Comment:
This is true! Mixins also allow us to inherit behavior from multiple modules, where as with inheritance, we can only inherit from one place (The Class we inherit from)